my-app/src/components/LoginPanel/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Button } from 'semantic-ui-react'
+import fire from '../Firebase/firebase'
+import Demo from './Demo';
+import { useTranslation } from 'react-i18next';
+import i18next from 'i18next';
+
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.login = this.login.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.signup = this.signup.bind(this);
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange(e) {
+    this.setState({ [e.target.name]: e.target.value });
+  }
+
+  login(e) {
+    e.preventDefault();
+    fire.auth().signInWithEmailAndPassword(this.state.email, this.state.password).then((u)=>{
+    }).catch((error) => {
+        console.log(error);
+      });
+  }
+
+  signup(e){
+    e.preventDefault();
+    fire.auth().createUserWithEmailAndPassword(this.state.email, this.state.password).then((u)=>{
+    }).then((u)=>{console.log(u)})
+    .catch((error) => {
+        console.log(error);
```

Comment:
shouldn't we inform user that something went wrong?

Reply:
done